src/queries/upsert_page_view.sql

```diff
@@ -0,0 +1,19 @@
+-- Function to increment the page view count for a given page
```

Comment:
Add a comment to make it clear that this is only for reference, since the function is stored in Supabase/Postgres itself

```suggestion
-- Only for reference, function is stored in Supabase/Postgres itself
-- Function to increment the page view count for a given page
```

Reply:
Add comment for SQL query